source/includes/fact-oplog-background.rst

```diff
@@ -1,7 +1,15 @@
-The :term:`oplog` in the source cluster must be large enough to track
-events that happen during the time it takes to complete the initial
-sync to the destination cluster.
+
+``mongosync`` applies operations in the ``oplog`` on the source cluster
+to the data on the destination cluster.  When older operations 
+that ``mongosync`` has not applied roll off the ``oplog`` 
+on the source cluster, the sync fails and ``mongosync`` exits.
+
+During initial sync, ``mongosync`` may apply operations at a slower
```

Comment:
Two comments: 
1. Probably good to include the article here. 
```suggestion
During the initial sync, ``mongosync`` may apply operations at a slower
```


2. Slower, faster, and more current are also a bit vague here. Is there any way to quantify these?

Reply:
Fixed point 1.  Leaving point 2 for the external reviewer.

Reply:
Fixed